.gitignore

```diff
@@ -4,13 +4,13 @@
 .vagrant/
 ansible/vagrant.retry
```

Comment:
I run "vagrant up", "vagrant ssh" etc from the root folder not from inside the ansible folder.. not sure how that got there